runtime/src/benchmarking/mnt_token.rs

```diff
@@ -8,33 +8,15 @@ runtime_benchmarks! {
 	{ Runtime, mnt_token }
 
 	_ {}
-	set_mnt_rate {
-		let pools = EnabledUnderlyingAssetsIds::get();
-		create_pools(&pools);
-		prepare_for_mnt_distribution(pools)?;
-		System::set_block_number(10);
-		MntToken::refresh_mnt_speeds()?;
-		System::set_block_number(11);
-	}: _(RawOrigin::Root, 15 * DOLLARS)
-
-	disable_mnt_minting {
-		let pools = EnabledUnderlyingAssetsIds::get();
-		create_pools(&pools);
-		prepare_for_mnt_distribution(pools)?;
-		System::set_block_number(10);
-		MntToken::refresh_mnt_speeds()?;
-		System::set_block_number(11);
-	}: _(RawOrigin::Root, BTC)
 
-	enable_mnt_minting {
+	set_speed {
 		let pools = EnabledUnderlyingAssetsIds::get();
 		create_pools(&pools);
 		prepare_for_mnt_distribution(pools)?;
 		System::set_block_number(10);
-		MntToken::refresh_mnt_speeds()?;
-		MntToken::disable_mnt_minting(RawOrigin::Root.into(), DOT)?;
+		MntToken::set_speed(RawOrigin::Root.into(), DOT, 0)?;
```

Comment:
```suggestion
		MntToken::set_speed(RawOrigin::Root.into(), DOT, 1)?;
```
When you set speed equals to zero, next time you ignore index updating.

Reply:
Done